src/posts/lucky_0.28.cr

```diff
@@ -0,0 +1,326 @@
+    * so many more...
+
+    ### No more avram_enum
+
+    When enums were first added to Avram, they were a bit tricky to implement. The
+    native Crystal enums didn't work without lots of refactoring. Since the time it
+    was first introduced, there's been enough refactoring to make adding native enums
+    a reality. Thanks to [Matthew McGarvey](https://github.com/matthewmcgarvey/), we
+    can delete a ton of code in our apps, and gain a lot flexibility!
+
+    Before this change, your enums would look like this:
+
+    ```crystal
+    class StreamEvent < BaseModel
+      avram_enum State do
+        Started
+        Waiting
+        Ended
+      end
+
+      table do
+        # NOTE: setting a default value here didn't work
+        column state : StreamEvent::State
+      end
+    end
+
+    # Saving the record
+    SaveStreamEvent.create!(state: StreamEvent::State.new(:started))
+
+    # Query for the record
+    StreamEventQuery.new.state(StreamEvent::State.new(:started).value)
+    ```
+
+    Now after the change, your enums look like this:
+
+    ```crystal
+    class StreamEvent < BaseModel
+      enum State
+        Started
+        Waiting
+        Ended
+      end
+
+      table do
+        # NOTE: now you can have default values
+        column state : StreamEvent::State = StreamEvent::State::Started
+      end
+    end
+
+    # Saving the record. It has a default value...
+    SaveStreamEvent.create!
+
+    # Query for the record
+    StreamEventQuery.new.state(StreamEvent::State::Started)
+    ```
+
+    ### New JSON serializable columns
+
+    Working with jsonb columns can be a bit tedious. Prior to this release, you
+    had to define your `JSON::Any`, and manually build out the structure you wanted.
+    With larger structures, you may have wanted to write some code like this:
+
+    ```crystal
+    table do
+      column data : JSON::Any
+    end
+
+    # This is easy to read, but comes at a huge performance cost...
+    JSON.parse(some_data_hash.to_json)
+
+    # This can get real messy real quick...
+    JSON::Any.new({"key" => JSON::Any.new(some_data_hash["key"])})
+    ```
+
+    Thankfully the [Crystal PG](https://github.com/will/crystal-pg/pull/232) shard now
+    has the ability to not parse the jsonb field from postgres immediately. This allowed
+    us to introduce serializable objects as columns! Now we can do this:
+
+    ```crystal
+    table do
+      column data : ReportData, serialize: true
+    end
+
+    class ReportData
+      include JSON::Serializable
+      property key : String
+    end
+    ```
+
+    Our jsonb columns now have more type-safety!
+
+    [read more](https://github.com/luckyframework/avram/pull/695) in the PR.
+
+    ### Support for subdomains
+
+    In your application, you may have sections of your site that require a subdomain
+    to access. For example, a separate API with `api.yoursite.com`, or maybe logging
+    in to the site sends you to a `dashboard.yoursite.com`.
```

Comment:
```suggestion
    Some applications are use multiple subdomains.
    For example, a separate API with `api.yoursite.com`, or maybe logging
    in to the site sends users to `dashboard.yoursite.com`.
```

Reply:
Ooo yup! I rewrote this too many times and it got away from me. I corrected my suggestion 😄